.rubocop.yml

```diff
@@ -6,12 +6,22 @@ EndOfLine:
   Enabled: false
 Layout/AlignParameters:
   EnforcedStyle: with_fixed_indentation
+Layout/BlockEndNewline:
+  Exclude:
+    - "spec/**/*"
```

Comment:
Why is spec excluded?

Reply:
Can't remember now but Rubocop reports no violations if I remove this exclusion.